src/viewport.js

```diff
@@ -940,8 +942,23 @@ $.Viewport.prototype = {
             this.degreesSpring.isAtTargetValue()) {
             return this;
         }
+       this.rotationPivot = pivot;
```

Comment:
This is great, but it would probably be good to follow the lead of `zoomPoint`: 

* Set it to `null` in the constructor, like so: https://github.com/openseadragon/openseadragon/blob/ed66c2d1518e919dffa335e68c34f469607273b3/src/viewport.js#L99
* Add some more checks before accepting it, like so: https://github.com/openseadragon/openseadragon/blob/ed66c2d1518e919dffa335e68c34f469607273b3/src/viewport.js#L889
* Reset it to null when the degreesSpring settles, like so: https://github.com/openseadragon/openseadragon/blob/ed66c2d1518e919dffa335e68c34f469607273b3/src/viewport.js#L1085

I don't mean to put the code in those places; that's just where you can see the `zoomPoint` equivalents.